wazo_call_logd/plugins/cdr/http.py

```diff
@@ -182,6 +187,41 @@ def visible_tenants(self, recurse=True):
         else:
             return [tenant_uuid]
 
+
+class RecordingsMediaResource(RecordingMediaAuthResource):
+    @required_acl(
+        'call-logd.cdr.recordings.media.delete',
+        extract_token_id=extract_token_id_from_query_or_header,
+    )
+    def delete(self):
+        args = RecordingMediaDeleteRequestSchema().load(request.get_json(force=True))
+        tenant_uuids = self.visible_tenants(True)
+        call_log_ids = args['cdr_ids']
+        for cdr_id in call_log_ids:
+            cdr = self.service.find_cdr(cdr_id, tenant_uuids)
+            if not cdr:
+                raise CDRNotFoundException(details={'cdr_id': cdr_id})
+
+        recordings = self.service.find_all_by(call_log_ids=call_log_ids)
+        for recording in recordings:
```

Comment:
Now there are in the same DB, you can change the logic
```suggestion
        for cdr_id in call_log_ids:
            cdr = self.service.find_cdr(cdr_id, tenant_uuids)
            if not cdr:
                raise CDRNotFoundException(details={'cdr_id': cdr_id})
            for recording in cdr.recordings:
```
(or something like this)

Reply:
Maybe adding a joinedload will be interesting for bulk delete on find_cdr